www/css/main.css

```diff
@@ -15799,7 +15945,16 @@ ion-content.padded-bottom-cta-with-summary {
       width: 100%; }
     .apl-amount-totals {
       margin-top: 14px;
-      padding: 0 16px; }
+      padding: 0 16px;
+      /*
+      &-title {
+        display: inline-block;
+      }
+      &-price {
+        //float: right;
```

Comment:
Comment to remove.

Reply:
We don't change the main.css, it is generated file.